nautobot/extras/forms.py

```diff
@@ -821,11 +785,7 @@ class StatusBulkEditForm(BootstrapMixin, CustomFieldBulkEditForm):
 
     pk = forms.ModelMultipleChoiceField(queryset=Status.objects.all(), widget=forms.MultipleHiddenInput)
     color = forms.CharField(max_length=6, required=False, widget=ColorSelect())
-    content_types = forms.ModelMultipleChoiceField(
-        queryset=ContentType.objects.filter(FeatureQuery("statuses").get_query()).order_by("app_label", "model"),
-        label="Content type(s)",
-        required=False,
-    )
+    content_types = MultipleContentTypeField(feature="statuses", required=False, label="Content type(s)")
```

Comment:
Bulk edit is experiencing the same issue:

![Editing_2_Statuses_-_Nautobot](https://user-images.githubusercontent.com/138052/112172370-e5742b00-8bb1-11eb-849a-7ae48f29864e.png)